docs/src/_cloud/onboarding.md

```diff
@@ -151,3 +143,8 @@ To view logs for any completed or still-running job, you can use:
 ```console
 meltano-cloud logs print --execution-id=ASDF1234...
 ```
+
```

Comment:
@cjohnhanson where should we have folks create a deployment? Likely before they run b/c `meltano-cloud run` requires the `--deployment` flag I believe.

Reply:
@tayloramurphy good callout, it's on my list of TODOs to add here before marking this PR as "ready". Just waiting on [the deployments CLI PR](https://github.com/meltano/meltano/pull/7718) to merge so I can be :100: confident on the commands and workflow.

Reply:
@tayloramurphy, just added this step. See latest changes: https://github.com/meltano/meltano/pull/7717/files#diff-08c41ed3fe9121e633df84a5c1e71fae5c2f11edf2726a8177e53d9e0ebdeab8R105